test/container/Dockerfile

```diff
@@ -93,8 +93,7 @@ COPY ./test/container/entrypoint.sh /usr/local/bin
 ARG UID
 
 # Prepare user configuration & build environments
-RUN userdel -r ubuntu && \
```

Comment:
Why remove this?

Reply:
That was an intermittent change, which I will revert. The issue is that when I try to build locally it tells me that email spool, or similar, is missing for user, and thus can not be deleted. 

I have commented about this issue here: https://github.com/argoproj/argo-cd/pull/18917#issuecomment-2211980621 

The PR was already closed sadly.

Reply:
Please, revert this change.

Reply:
I reverted similar change in wrong file. I have now reverted the change and squashed the commits.